IFC4x3/Properties/m/MaximumSolarRadiation_3GwIZXL0151QO$6Rwbh8Tj/Documentation.md

```diff
@@ -1 +1 @@
-Maximum level of solar radiation that the equipment complies with, usually measured by duration of solar radiation.
+Maximum level of solar radiation that the equipment complies with. The value indicates power density measured in watt per square meter (w/m2).
```

Comment:
```suggestion
Maximum level of solar irradiance that the equipment can operate effectively in. This is usually tested and measured by a national or international standard. The value indicates power density measured in watt per square meter (w/m2).
```
suggest name change to `MaximumSolarIrradiance` which is the correct scientific naming for the intended process being described.